sympy/polys/polytools.py

```diff
@@ -5202,6 +5202,17 @@ def try_non_polynomial_gcd(seq):
 
     try:
         polys, opt = parallel_poly_from_expr(seq, *gens, **args)
+
+        #gcd for domain Q[irrational] (purely algebraic irrational)
```

Comment:
a single space after # is the correct way refer [this](https://www.python.org/dev/peps/pep-0008/#inline-comments)

Reply:
Thank you for the review. Made the changes.